vault/programs/vault/src/context.rs

```diff
@@ -0,0 +1,323 @@
+use crate::zeta_context::*;
+use crate::*;
+
+#[derive(Accounts)]
+#[instruction(vault_name: String, vault_lamports: u64, bumps: VaultBumps)]
+pub struct InitializeVault<'info> {
+    // vault Authority accounts
+    #[account(mut)]
+    pub vault_authority: Signer<'info>,
```

Comment:
Can anyone create a vault?

Reply:
Yes, any reason why we wouldn't want that?

Reply:
Seems fine - but I would feel the vault protocol would want to launch their own exclusively.